receiver/elasticapmreceiver/internal/testutil/testutil.go

```diff
@@ -0,0 +1,135 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
```

Comment:
Copy of https://github.com/open-telemetry/opentelemetry-collector/blob/main/internal/testutil/testutil.go